### ISSUE TITLE
Update to Linux Mint 22

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/cinnamon#115]
```bash
sudo apt update && sudo apt upgrade -y && sudo apt autoremove -y && sudo apt dist-upgrade -y
```

does not help.

What about `mintupdate`?

**Comment by Benjamin-Loison** [comment]
Related to [Benjamin-Loison/mintupdate/issues/2](https://github.com/Benjamin-Loison/mintupdate/issues/2).

**Comment by Benjamin-Loison** [comment]
https://blog.linuxmint.com/?p=4629

**Comment by Benjamin-Loison** [comment]
```bash
mint-upgrade-info
```
```
mint-upgrade-info: command not found
```

**Comment by Benjamin-Loison** [comment]
https://blog.linuxmint.com

I subscribed to new posts thanks to https://blog.linuxmint.com/?feed=rss2and OverClock3000 `notifyOnRSSUpdate.py`.

**Comment by Benjamin-Loison** [comment]
https://forums.linuxmint.com/viewtopic.php?t=416123
https://forums.linuxmint.com/viewtopic.php?t=423049
https://forums.linuxmint.com/viewtopic.php?t=425073

**Comment by Benjamin-Loison** [comment]
Closed in favor of #116.
